public/app/features/alerting/unified/components/rule-list/EvaluationGroup.tsx

```diff
@@ -0,0 +1,117 @@
+import { css, cx } from '@emotion/css';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, Badge, Button, Dropdown, Menu, Stack, Text, Icon } from '@grafana/ui';
+
+import { MetaText } from '../MetaText';
+import { Spacer } from '../Spacer';
+
+interface EvaluationGroupProps extends PropsWithChildren {
+  name: string;
+  description?: ReactNode;
+  interval?: string;
+  provenance?: string;
+  isOpen?: boolean;
+  onToggle: () => void;
+}
+
+const EvaluationGroup = ({
+  name,
+  description,
+  provenance,
+  interval,
+  onToggle,
+  isOpen = false,
+  children,
+}: EvaluationGroupProps) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.groupWrapper} role="treeitem" aria-expanded={isOpen} aria-selected="false">
```

Comment:
should we also add [aria-checked](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked)?

Reply:
could we use a `Stack` here and remove styles?

Reply:
> could we use a Stack here and remove styles?

Yes! 🥳